tests/foreman/ui/test_ansible.py

```diff
@@ -519,6 +519,55 @@ def test_positive_assign_and_remove_ansible_role_to_hostgroup(
             session.hostgroup.delete(name)
             assert not target_sat.api.HostGroup().search(query={'search': f'name={name}'})
 
+    @pytest.mark.tier3
+    def test_positive_non_admin_hostgroup_permission(
+        self, request, function_org, function_location, target_sat
+    ):
+        """Verify Hostgroup page is accessible to non-admin user.
+
+        :id: 3b8ce895-9d4c-491d-a5eb-d292748134a5
+
+        :customerscenario: true
+
+        :BZ: 2236418
+
+        :steps:
+            1. Create a user with non-admin permissions and assign "Ansible Roles Manager", "Ansible Tower Inventory Reader"
+            and "Content Importer" to the user.
```

Comment:
You mentioned "Content Importer" to the user, but it is not included in the role. You can either remove it from your steps or add it to the role  below.

Reply:
Removed from docstring.